src/helpers.rs

```diff
@@ -85,6 +87,39 @@ pub fn create_client_local() -> Result<TonClient, String> {
     Ok(Arc::new(cli))
 }
 
+lazy_static! {
+    static ref MAIN_ENDPOINTS: Vec<String> = vec![
+        "main2.ton.dev".to_string(),
+        "main3.ton.dev".to_string(),
+        "main4.ton.dev".to_string(),
+    ];
+
+    static ref NET_ENDPOINTS: Vec<String> = vec![
+        "net1.ton.dev".to_string(),
+        "net5.ton.dev".to_string(),
+    ];
+}
+
+// TODO: Organize endpoints to list in the external resource
+fn resolve_endpoints(url: &str) -> Option<Vec<String>> {
+    macro_rules! network_test {
+        ($name:literal, $url:expr) => (
+            Regex::new(&format!(r"^\s*(https?://)?{}\.ton\.dev\s*", $name))
+                .expect("Regex compilation error")
```

Comment:
Can you create Regex instance only once? Like 
`let regex = Regex::new().ok()?;`
And then use it twice.
`regex.is_match(url)`;